pkg/restapi/operation/operations_test.go

```diff
@@ -319,7 +329,7 @@ func TestHandleOIDCCallback(t *testing.T) {
 					oidcClaimsFunc: func(v interface{}) error {
 						c, ok := v.(*oidcClaims)
 						require.True(t, ok)
-						c.Sub = sub
+						c.Sub = id
```

Comment:
@llorllale I think these were missed in the last PR - let me know if you intended for these to be `sub` still